api/v1alpha1/proxmoxmachine_types.go

```diff
@@ -216,6 +216,13 @@ type NetworkDevice struct {
 	// +kubebuilder:validation:Enum=e1000;virtio;rtl8139;vmxnet3
 	// +kubebuilder:default=virtio
 	Model *string `json:"model,omitempty"`
+
+	// MTU is the network device Maximum Transmission Unit.
+	// Only works with virtio Model.
+	// +optional
+	// +kubebuilder:validation:Minimum=1
```

Comment:
minimum=1 is okay,
but you can set value like 100

we probably have to check this in the webhook.

Reply:
`Should not allow mtu less than 1000 and more than 65520`

Reply:
I think less than 1000 should be allowed because 1 equals the MTU of the network bridge: 

https://git.proxmox.com/?p=qemu-server.git;a=commit;h=61a14cde8d568e552d3deaab2da76b479b8aca7b
`Special value: 1 = set the same value than the bridge`

Reply:
Ah okay now I get it, I will try to implement that. Thanks for the review!

Reply:
Thank you for addressing this. 💯